integrated_channels/degreed2/client.py

```diff
@@ -114,6 +114,7 @@ def create_course_completion(self, user_id, payload):
         Returns: status_code, response_text
         """
         json_payload = json.loads(payload)
+        json_payload['data']['attributes']['percentile'] = json_payload['data']['attributes']['percentile'] * 100
```

Comment:
🤔 Wondering if this can go in the exporter

Reply:
good suggestion - I think it's a little cleaner moving this into the exporter as well!